app/routes/main/athleteDetails/selector.js

```diff
@@ -1,10 +1,18 @@
+import idx from "idx";
+
 import { getCurrentUserID } from "../../../store/state/appState/selectors";
 import { getEntity } from "../../../store/state/entities/selectors";
 
+const getAthletePerformances = props => idx(props, _ => _.performance.details);
```

Comment:
may be should consider storing it like an entity?

Reply:
i feel like we are the right track here

Reply:
Entity? May be! Right now: don't think so! Still be considered..

Reply:
```
render() {
    const { performances } = this.props;
    if (!this.props.rendered) return false;

    const unit=detail.unit !== undefined
      ? detail.unit
      : ""

    const performanceList = performances.map((detail, index) => ({
      key: index,
      image: {
        name: getIconName(detail.name),
        color: theme.PrimaryColor
      },
      text: `${detail.value}  ${unit}`
    }));

    return (
      <CardList
        title="DETAILS"
        list={performanceList}
      />
    );
  }
```

Reply:
oups units should be inside the map